internal/blueprint/blueprint_test.go

```diff
@@ -0,0 +1,136 @@
+// Package blueprint contains primitives for representing weldr blueprints and
```

Comment:
Really tiny nit: is the repetition of the comment necessary to make the linter happy?

Reply:
No idea, the file was automatically generated as we discussed it.

Reply:
Ah, then leave it :)